themes/default/css/wide.css

```diff
@@ -1226,7 +1226,7 @@ select.objectRepresentationOverlayControls {
 }
 
 #objectInfoRepresentationOverlay div.close {
-	background:url(../graphics/buttons/close.png) no-repeat; 
+	background:url(../graphics/icons/glyphicons_192_circle_remove.png) no-repeat;
```

Comment:
We should replace these icons with Font-Awesome icons (using caNavIcon()?)

Reply:
I will remove this commit, it shouldn't be in this PR.